test/e2eTests/pages/page.js

```diff
@@ -10,16 +14,25 @@ const translator = require('./../translator');
  * This class adds the component's properties
  * to the class instance which lets tests call component methods
  * on the instance as if they were one object. See Compoent for more details.
+ *
  */
+
 class Page {
   constructor(options) {
+
+    if(!options) {
+      options={isLoaded:{ urlPart:"", element: ""}};
```

Comment:
Thanks. We can have it throw an error/warning, but I'm of the mind it should always initialize to sane defaults, it also provides the template for the expected json object (e.g. in case some new developer doesn't know to set something). We should probably make this initialization more robust and flexible than what I've added.